server/zio-http/src/test/scala/sttp/tapir/server/ziohttp/ZioHttpServerTest.scala

```diff
@@ -0,0 +1,32 @@
+package sttp.tapir.server.ziohttp
+
+import cats.effect.{IO, Resource}
+import sttp.capabilities.zio.ZioStreams
+import sttp.monad.MonadError
+import sttp.tapir.server.tests.{
+  DefaultCreateServerTest,
+  ServerAuthenticationTests,
+  ServerBasicTests,
+  ServerMetricsTest,
+  ServerStreamingTests,
+  backendResource
+}
+import sttp.tapir.server.zhttp.ZioHttpInterpreter.zioMonadError
+import sttp.tapir.tests.{Test, TestSuite}
+import zio.RIO
+import zio.blocking.Blocking
+
+class ZioHttpServerTest extends TestSuite {
+
+  override def tests: Resource[IO, List[Test]] = backendResource.map { backend =>
+    val interpreter = new ZioHttpTestServerInterpreter()
+    val createServerTest = new DefaultCreateServerTest(backend, interpreter)
+
+    implicit val m: MonadError[RIO[Blocking, *]] = zioMonadError
+
+    new ServerBasicTests(createServerTest, interpreter, false, true, false, false).tests() ++
```

Comment:
[minor] it's good practice to name boolean parameters. I think IntelliJ has an inspection for that as well